app/controllers/api/v1/transactions_controller.rb

```diff
@@ -0,0 +1,45 @@
+module Api::V1
+  class TransactionsController < OrganizationController
+    before_action :set_transaction,  only: [:show, :edit, :update, :destroy]
+
+    def index
+      @transactions = current_organization.transactions.page(params[:page]).per(30)
+    end
+
+    def show
+    end
+
+    def create
+      @transaction = current_organization.bank_accounts.find_by(id: transaction_params[:bank_account_id]).transactions.build transaction_params
+      @transaction.created_by = current_user
+
+      if @transaction.save
```

Comment:
@yunixon instead render `create` template (by default) render `show` template`. They are absolutely identical (and they should be identical). Same for update